updated tests guideline (#4840)
kyma-project/kyma: tests/README.md

Every Kyma test resides in a dedicated folder which contains its sources and a `README.md ` file. This file provides instructions on how to build and develop the test suite.

The test name, which is also the folder name, is the component's name without any prefix or suffix. For example, `apiserver-proxy`.

> The Docker image resulting from the sources of a test suite resides in the regular component folder having a suffic indicating the testing nature, like `-integration-tests`.

Reviewer: ```suggestion
The Docker image created from the sources of a test suite resides in a component folder marked with a suffix indicating the testing nature, such as `-integration-tests`.
```
